src/main/scala/com/monsanto/arch/cloudformation/model/simple/Builders.scala

```diff
@@ -203,18 +203,20 @@ trait Autoscaling {
     userData:     `Fn::Base64`,
     iam:          Option[Token[ResourceRef[`AWS::IAM::InstanceProfile`]]] = None,
     condition:    Option[ConditionRef] = None,
-    dependsOn:    Option[Seq[String]]  = None
+    dependsOn:    Option[Seq[String]]  = None,
+    blockDevices: Option[Seq[BlockDeviceMapping]] = None
```

Comment:
I think they do use named parameters now in microservices-vpc, but it's probably better to keep backwards compatibility.